box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxArray.java

```diff
@@ -0,0 +1,43 @@
+package com.box.androidsdk.content.models;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * A collection that contains BoxJsonObject items
+ *
+ * @param <E> the type of elements in this partial collection.
+ */
+public class BoxArray<E extends BoxJsonObject> extends BoxList<E> {
+
+    public static final String PUT_ARRAY = "put_array";
+
+    protected final Collection<E> collection = new ArrayList<E>();
+
+    public BoxArray() {
+        super();
+        mProperties.put(FIELD_ENTRIES, collection);
+    }
+
+    /**
+     * Constructs a BoxList with the provided map values.
+     *
+     * @param map map of keys and values of the object.
+     */
+    public BoxArray(Map<String, Object> map) {
+        super(map);
+    }
+
+    @Override
+    public String toJson() {
+        StringBuffer json = new StringBuffer("[");
```

Comment:
We shouldn't be building the JSON manually -- look into the other toJson methods and construct JsonArrays and JsonObjects before going toString on them